src/static/js/corsclient.js

```diff
@@ -606,7 +606,8 @@ var sendRequest = function(controller, url) {
 
   logger.log(msg);
 
-  xhr.send();
+  xhr.send(controller.getValue('client_postdata'));
```

Comment:
Small update: Can this only call the send(data) version if client_postdata has actual content?